deployment/src/main/java/io/quarkiverse/jackson/jq/deployment/JacksonJqFunctionLoader.java

```diff
@@ -0,0 +1,97 @@
+package io.quarkiverse.jackson.jq.deployment;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Enumeration;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.MappingIterator;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+import net.thisptr.jackson.jq.Scope;
+import net.thisptr.jackson.jq.Version;
+import net.thisptr.jackson.jq.VersionRange;
+import net.thisptr.jackson.jq.internal.misc.VersionRangeDeserializer;
+
+public class JacksonJqFunctionLoader {
```

Comment:
The only "change" that would be needed at this point for the jackson-jq library would be to expose a public way to de-serialize the jq.json file: do you think it would be possible  ?